src/include/iscsi-lio-server/TargetData.rb

```diff
@@ -1,4 +1,26 @@
+# encoding: utf-8
+
+# Copyright (c) [2017] SUSE LLC
```

Comment:
NP: s/2017/2018

Or... 2019 ;)

Reply:
As commented in IRC, I used the year when the file was created.

Reply:
But then you touched it in the current year. :wink: